toolchain/parser/parser.cpp

```diff
@@ -17,6 +17,18 @@
 
 namespace Carbon {
 
+static auto ToString(Parser::ParseContext parse_context) -> std::string {
+  switch (parse_context) {
+    case Parser::ParseContext::File:
+      return "file";
+
+    case Parser::ParseContext::Interface:
+      return "interface";
+
+    case Parser::ParseContext::Class:
+      return "class";
+  }
+}
```

Comment:
```suggestion
}

```

Please keep a blank line between bodies.